ena/ERC000030.json

```diff
@@ -0,0 +1,187 @@
+        "depth": {
+          "type": "string",
+          "pattern": "^-?\d{1,4}\.?\d{0,2}$"
+        },
+        "sampleCollectionDevice": {
+          "type": "string"
+        },
+        "protocolLabel": {
+          "type": "string"
+        },
+        "sizeFractionLowerThreshold": {
+          "type": "string"
+        },
+        "sizeFractionUpperThreshold": {
+          "type": "string"
+        },
+        "sampleStatus": {
+          "type": "string"
+        },
+        "lastUpdateDate": {
+          "type": "string",
+          "format": "date"
+        }
+      }
+    },
+    "nonSampleTerms": {
+      "type": "object",
+      "required": [
+        "projectName"
+      ],
+      "properties": {
+        "projectName": {
+          "type": "string"
+        }
+      }
+    },
+    "collectionEventInformation": {
+      "type": "object",
+      "required": [
+        "environmentalPackage",
+        "environmentBiome",
+        "environmentFeature",
+        "environmentMaterial"
+      ],
+      "properties": {
+        "environmentalPackage": {
+          "type": "string",
+          "enum": [
+            "Air", "Built-environment", "Host-associated", "Human-associated",
+            "Human-gut", "Human-oral", "Human-skin", "Human-vaginal",
+            "Microbial mat/biofilm", "Miscellaneous natural or artificial environment",
+            "Plant-associated", "Sediment", "Soil", "Wastewater/sludge",
+            "Water"
+          ]
+        },
+        "environmentBiome": {
+          "type": "string"
+        },
+        "environmentFeature": {
+          "type": "string"
+        },
+        "environmentMaterial": {
+          "type": "string"
+        }
+      }
+    },
+    "marineSampling": {
+      "type": "object",
+      "required": [
+        "samplingCampaign",
+        "samplingStation",
+        "samplingPlatform"
+      ],
+      "properties": {
+        "samplingCampaign": {
+          "type": "string"
+        },
+        "samplingStation": {
+          "type": "string"
+        },
+        "samplingPlatform": {
+          "type": "string"
+        }
+      }
+    },
+    "marineEnvironment": {
+      "type": "object",
+      "required": [
+        "marineRegion",
+        "salinitySensor",
+        "temperature"
+      ],
+      "properties": {
+        "marineRegion": {
+          "type": "string",
+          "enum": []
+        },
+        "salinitySensor": {
+          "type": "string",
+          "pattern": "^\d{1,2}\.?\d{0,4}$"
+        },
+        "oxygenSensor": {
+          "type": "string",
+          "pattern": "^\d{1,3}\.?\d{0,4}$"
+        },
+        "nitrateSensor": {
+          "type": "string",
+          "pattern": "^\d{1,2}\.?\d{0,4}$"
+        },
+        "temperature": {
+          "type": "string",
+          "pattern": "^\d{1,2}\.?\d{0,4}$"
+        },
+        "chlorophyllSensor": {
+          "type": "string",
+          "pattern": "^\d{1,2}\.?\d{0,4}$"
+        }
+      }
+    },
```

Comment:
Though it may be a valid json schema, this whole `properties` section doesn't match the USI sample structure.
None of these properties are part of the model and as such they need to be represented as `attributes` and they need to match the USI attributes structure.
An excerpt to exemplify this:
```js
(...)
"attributes": {
    "description": "Attributes for describing a sample.",
    "type": "object",
    "properties": {
        "marineEventLabel": {
            "description": "An Event corresponds to the deployment of a sampling device. It is bound in space and time. The Event Label is a unique identifier composed of the project prefix \"TARA\", sampling date/time \"yyyymmddThh:mm:ssZ\", station number \"000\" and a short label to indicate the sampling device used, e.g. \"EVENT_CAST\", \"EVENT_NET\", \"EVENT_PUMP\". Example: TARA_20110416T1508Z_100_EVENT_CAST.",
            "maxItems": 1,
            "items": {
                "properties": {
                    "value": {
                        "pattern": "^TARA\w*"
                    }
                }
            }
        },
        "marineEventDateTimeStart": {
            "description": "Date and time in UTC when the sampling event started, e.g. each CTD cast, net tow, or bucket collection is a distinct event. Format: yyyy-mm-ddThh:mm:ssZ. Example: 2011-04-16T15:05:30Z",
            "maxItems": 1,
            "items": {
                "properties": {
                    "value": {
                        "format": "date-time"
                    }
                }
            }
        }
    },
    "patternProperties": {
        "^.*$": { 
            "type": "array",
            "minItems": 1,
            "items": {
                "properties": {
                    "value": { "type": "string", "minLength": 1 },
                    "units": { "type": "string", "minLength": 1 },
                    "terms": {
                        "type": "array",
                        "items": {
                            "type": "object",
                            "properties": {
                                "url": {"type": "string", "format": "uri" }                    
                            },
                            "required": ["url"]
                        }
                    }
                },
                "required": ["value"]
            }
        }
    },
    "required": ["marineEventDateTimeStart"]
}
(...)
```
Note that besides this last `required` section, the attributes must also be added to the root object required section (together with alias, taxonId and releaseDate).

Reply:
An example sample that complies with the complete version of this schema:
```js
{
	"alias": "abc",
	"taxonId": 9606,
	"releaseDate": "2018-01-01",
	"attributes": {
		"marineEventLabel": [{
			"value": "TARA_20110416T1508Z_100_EVENT_CAST"
		}],
		"marineEventDateTimeStart": [{
			"value": "2011-04-16T15:05:30Z"
		}]
	}
}
```